presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/EnvironmentUp.java

```diff
@@ -145,7 +145,9 @@ public Integer call()
                 builder.configureContainers(Standard::enablePrestoJavaDebugger);
             }
 
+            environmentConfig.extendEnvironment(this.environment).ifPresent(extender -> extender.extendEnvironment(builder));
```

Comment:
Can you extend it to separate PR. Also maybe rename `extendEnvironment` to getEnvironmentExtender.